build/commands/lib/perfTests.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+const util = require('../lib/util')
+const config = require('../lib/config')
+const path = require('path')
+
+
+const runPerfTests = (passthroughArgs, perf_config, targets, options = {}) => {
```

Comment:
It looks like `options` is not being used.

Reply:
Fixed.